pkg/antctl/command_definition_test.go

```diff
@@ -38,8 +38,10 @@ import (
 	"antrea.io/antrea/pkg/antctl/transform/common"
 	"antrea.io/antrea/pkg/antctl/transform/controllerinfo"
 	"antrea.io/antrea/pkg/antctl/transform/networkpolicy"
+	"antrea.io/antrea/pkg/apis/controlplane/v1beta2"
```

Comment:
Duplicate import with following one.
As quan suggested, you need to check the result of github action result. For golint issue, you can also run `make golangci` locally to check this kind of issue.

Reply:
Fixed